README.md

```diff
@@ -0,0 +1,11 @@
+# Indigent Defense Azure Scraper
+
+(and eventually parser.)
+
+### Steps to run locally
+
+- Put environment variables in `local.settings.json` (ask an Open Austin dev for this info)
```

Comment:
I think we might be able to set permissions between the container and the app so we don't have to store credentials locally